client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -278,6 +303,22 @@ export const siteSetupFlow: Flow = {
 				}
 
 				case 'vertical': {
+					const goalsCaptureStepEnabled = isEnabled( 'signup/goals-step' );
+
+					if ( goalsCaptureStepEnabled ) {
+						if ( goals.includes( SiteGoal.Import ) ) {
+							return navigate( 'import' );
+						}
```

Comment:
Approaching Goals Capture i1 (iteration 1) as a direct drop-in replacement of the Intent step, it would be preferable to handle this as an intent instead.